jena-fuseki2/jena-fuseki-ui/vue.config.js

```diff
@@ -23,6 +24,34 @@ module.exports = {
         args[0].title = 'Apache Jena Fuseki'
         return args
       })
+
+    if (process.env.NODE_ENV !== 'production') {
+      // devtool for test and other modes
+      // https://webpack.js.org/configuration/devtool/
+      if (process.env.NODE_ENV === 'test') {
+        // NOTE: if you need to debug the project with WebStorm (or another IDE) and it fails, try
+        //       change this value for config.devtool('eval-source-map')
+        config.devtool('eval')
+      } else {
+        config.devtool('eval-source-map')
+      }
+
+      // coverage
+      if (process.env.NODE_ENV === 'test') {
+        config.module.rule('istanbul')
+          .test(/\.js$/)
+          .include.add(path.resolve('src')).end()
+          .use('istanbul-instrumenter-loader')
+          .loader('istanbul-instrumenter-loader')
+          .options({ esModules: true })
+          .after('cache-loader')
+      }
+
+      // resolve modules in devtool
+      config.output
+        .devtoolModuleFilenameTemplate('[absolute-resource-path]')
+        .devtoolFallbackModuleFilenameTemplate('[absolute-resource-path]?[hash]')
+    }
```

Comment:
This is a mix from some configs from docs about [setting up coverage](https://gist.github.com/lsapan/3bfd0ffc0fb3d4a036fce84f6eea142e) with some copy-paste from a previous project I worked on.